VideoLocker/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -241,4 +257,26 @@ public void onActivityStarted(Activity activity) {
         public void onActivityStopped(Activity activity) {
         }
     }
+
+    public void clearWebViewCache(){
+        try {
+            boolean success = deleteDatabase("webview.db");
```

Comment:
Are these set up by the system? Is it possible to read these paths from the android libraries? I'm just imagining them deciding to change the path or it being different in different languages or something.